lib/sailci.js

```diff
@@ -0,0 +1,17 @@
+// http://docs.sail.ci/documentation/environment-variables
+
+module.exports = {
+	detect({env}) {
+		return Boolean(env.SAILCI);
+	},
+	configuration({env}) {
+		return {
+			name: 'Sail CI',
+			service: 'sailci',
```

Comment:
For consistency reasons could you set the `service` to `sail` rather than `sailci`?